src/textarea/textarea.tsx

```diff
@@ -9,25 +9,127 @@ interface Props {
     className?: string;
     placeholder?: string;
     disabled?: boolean;
+    isCharacterLimitEnabled?: boolean;
 }
+interface State {
```

Comment:
It seems like multiple things here are never changed. Why are the in state? `radius`, `minRows` ... stopping here. Please check.

Reply:
fixed